lib/dep-graph-builders/yarn-lock-v1/workspaces.ts

```diff
@@ -11,15 +11,14 @@ export const parseYarnLockV1WorkspaceProject = async (
   workspacePackagesPkgJsons: string[],
   options: ProjectParseOptions,
 ): Promise<DepGraph[]> => {
-  const { includeDevDeps, includeOptionalDeps, pruneCycles, strictOutOfSync } =
-    options;
+  const {
+    includeDevDeps,
+    includeOptionalDeps,
```

Comment:
This one needs to be removed?

Reply:
Will be fixed soon, we need to handle it later now